wsd/DocumentBroker.cpp

```diff
@@ -430,8 +428,7 @@ void DocumentBroker::pollThread()
         if (isLoaded() && getIdleTimeSecs() >= IdleDocTimeoutSecs)
         {
             // Don't hammer on saving.
-            if (std::chrono::duration_cast<std::chrono::seconds>(now - _lastSaveRequestTime).count()
-                >= 5)
+            if (_saveManager.timeSinceLastSaveRequest() >= std::chrono::seconds(5))
```

Comment:
I wonder - at one occasion there is a specific check method (hasSavingTimedOut()), yet here we expose the time, and check explicitly in the if here; looks a bit inconsistent (?)

Reply:
I didn't want to change the logic in this PR. The helper you mention checks that there is a request active, here the comparison is unconditional regardless of the request finished or not.

In a later patch this logic will be reworked to centralize the decision to save and upload.